force-app/main/default/classes/UnitOfWork.cls

```diff
@@ -41,12 +41,14 @@ public inherited sharing class UnitOfWork {
 
     public Database.SaveResult[] save() {
         List<Database.SaveResult> saveResults = new List<Database.SaveResult>();
+
```

Comment:
If there are no changes to this file could it be reverted so that these diff lines go away?